examples/adb-with-private-link-standard/main.tf

```diff
@@ -1,5 +1,5 @@
 module "adb-with-private-link-standard" {
-  source       = "github.com/databricks/terraform-databricks-examples/modules/adb-with-private-link-standard"
```

Comment:
I'm not sure about this - if people will copy the code without cloning the repo, then use of the github is better

Reply:
sure, I wasn't sure about preferences here, will revert